docs/upgrades/locks/l0ckbox.mdx

```diff
@@ -37,3 +55,11 @@ Connection terminated.
 ```
 
 </details>
+
+## l0ckbox
+
+<details>
+  <summary>Spoilers for l0ckbox's solutions</summary>
+Load the correct [[k3y_v1]] or [[key_v2]] upgrade which has the same k3y value that the lock requires.
+
+</details>
```

Comment:
```suggestion
```
This section is normally used for named parameters for the lock (e.g. the `ez_40` and `ez_prime` parameters on the `ez_40` page). The behavior described here is also already documented in the "Behavior" section above.

Otherwise, looks good to me.